recipes/gklib/all/conandata.yml

```diff
@@ -0,0 +1,10 @@
+sources:
+  "5.1.1":
+    url: "https://github.com/KarypisLab/GKlib/archive/refs/tags/METIS-v5.1.1-DistDGL-0.5.tar.gz"
+    sha256: "52aa0d383d42360f4faa0ae9537ba2ca348eeab4db5f2dfd6343192d0ff4b833"
+patches:
+  "5.1.1":
+    - patch_file: "patches/001-install-win32-headers.patch"
```

Comment:
Does the newer toolchain allow use to skip the base path that was previously required? 🤯

Reply:
from https://blog.conan.io/2021/10/19/New-CMake-Transparent-Integration-Guide.html

> Also the new tool apply_conandata_patches already knows where to locate the sources thanks to the layout, so the base_path can be omitted in the conandata.yml file.

Reply:
🚀  thanks for the link!